README.md

```diff
@@ -202,6 +202,91 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
 
     > **Note:** that every minute on the clock a job is triggered to reconcile the CRs with the chart values. These cronjobs are deleted when their respective features (e.g. `rhdhOperator.enabled=false`) are removed or when the chart is removed. This is required because the CRs are not managed by helm due to the CRD dependency pre availability to the deployment of the CR.
 
+### Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, ie: not in-memory.
+
+Kafka perfectly fullfil this reliability need.
+
+For that, you will need:
+
+1. A Kafka cluster running, see https://strimzi.io/quickstarts/ for a quickstart setup
+1. To configured and enable kafka broker feature in Knative:
```

Comment:
Although it is rendered nicely, pls change `1.` to `2.` for consistency.

Reply:
Ah, ok, I did that for consistency: https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-chart/gh-pages/README.md => there is
```
1. Deploy the PostgreSQL reference implementation for persistence support in SonataFlow following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/postgresql/README.md)

1. Create a namespace for the Orchestrator solution:

   ```console
   oc new-project orchestrator
   ```

1. Create a namespace for the Red Hat Developer Hub Operator (RHDH Operator):

   ```console
   oc new-project rhdh-operator
   ```


1.  Download the setup script from the github repository and run it to create the RHDH secret and label the GitOps namespaces:
...
```

Reply:
+1 for consistency, but in this section it is like:
```
1. ...
1. ...
3. ...
4. ...
```

so either all be `1....` or numbered.